manuscript.Rmd

```diff
@@ -302,8 +308,95 @@ The cloud is the natural environment of containers, and becomes the go-to mechan
 - [`googleComputeEngineR`](https://cloudyr.github.io/googleComputeEngineR/) (function `docker_run`)
   - I don't think `docker_run` is main focus for containers for this package, its more how it uses Docker to launch custom environments that can be templated (e.g. rstudio, shiny etc.) and for parrallisation e.g. `library(future)`  using GCE VMs as endpoints. [@MarkEdmondson1234]
 - [`analogsea`](https://github.com/sckott/analogsea) (digital ocean R client)
-- [`plumber`](https://www.rplumber.io/docs/hosting.html#docker)
-- [production deployment of neural networks](https://github.com/tmobile/r-tensorflow-api) [@jnolis, @nolistic]
+- [`AzureContainers`](https://cran.r-project.org/package=AzureContainers): Umbrella package for working with containers in Microsoft's Azure cloud. Provides interfaces to three Azure services: Container Registry, Container Instances (for running individual containers), and Kubernetes Services (for orchestrated deployments)
+
+## Using R to power enterprise software in production environments [@jnolis, @nolistic]
+
+R has been historically viewed as a tool for analysis and scientific research--not for creating software that corporations can rely on to continously run in real time. However, thanks to advancements in R running as a web service, along with with the ability to deploy R in docker containers, modern enterprises are now capable of having real-time machine learning powered by R.
+
+The backbone of this work is the _[plumber](https://www.rplumber.io/docs/hosting.html#docker)_ package, created by Jeff Allen. The plumber package allows R to be used to host web services through RESTful APIs. When R is running on a server, a different server can ask R to make a computation through an HTTP request and get an immediate response. For instance, if a data scientist at a company creates a machine learning model to tell which customers are likely to make a repeat purchase, if they serve the model through a plumber API then someone else on the marketing team could write software that sends different emails depending on that real-time prediction. Plumber syntax is simple to read and understand: the API is created by commenting above function names in an R script.
```

Comment:
We have to keep an eye on the level of detail, and to me this goes a bit beyond the information about plumber that we can squeeze in. I'll try to shorten it.